### PR TITLE
Added help dialog for shortcuts

### PR DESCRIPTION
**Pull request by Dev-Mikko**
Added an help button for showing all the shortcut keys enabled in the client.

immagine

immagine

immagine

### CODE REVIEW

**Comment by Dev-Mikko** [comment]
@Dev-Mikko OK, thank you. As you can see, I have already fixed some merge conflicts with main branch.

**Review by pnts-se — CHANGES_REQUESTED** [review]
Added comments related to naming.

File: src/components/menuitems/HelpMenuItem.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { MenuItem } from '@mui/material';
+import { Help } from '@mui/icons-material';
```
[the remark this answers is not in the document]
Comment (Dev-Mikko) [reply src/components/menuitems/HelpMenuItem.tsx:2]: Done

File: src/components/topbar/TopBar.tsx [hunk 2]
```diff
@@ -47,6 +47,8 @@ import Filesharing from '../menuitems/Filesharing';
 import TranscriptionButton from '../controlbuttons/TranscriptionButton';
 import Transcription from '../menuitems/Transcription';
 import { AccessTime } from '@mui/icons-material';
+import HelpButton from '../controlbuttons/HelpButton';
+import HelpMenuItem from '../menuitems/HelpMenuItem';
```
[the remark this answers is not in the document]
Comment (Dev-Mikko) [reply src/components/topbar/TopBar.tsx:51]: Done

**Comment by Dev-Mikko** [comment]
@pnts-se fixed as you said 😄

**Pull request merged**